packages/backend-core/src/features/index.ts

```diff
@@ -1,24 +1,91 @@
 import env from "../environment"
 import * as context from "../context"
 import { cloneDeep } from "lodash"
+import { PostHog, PostHogOptions } from "posthog-node"
+import { IdentityType } from "@budibase/types"
+import tracer from "dd-trace"
+
+let posthog: PostHog | undefined
+export function init(opts?: PostHogOptions) {
+  if (env.POSTHOG_TOKEN) {
+    posthog = new PostHog(env.POSTHOG_TOKEN, {
+      host: "https://us.i.posthog.com",
```

Comment:
We should have this string as env (maybe with a default value), just in case something changes from their end